pbj-core/pbj-runtime/src/test/java/tests/ComparableOneOfTest.java

```diff
@@ -28,7 +29,8 @@ void asReturnsValue() {
     @Test
     void hashCodeReturnsHashCode() {
```

Comment:
I suggest to implement this as a parameterized tests that checks all values of `TestEnum`.

Reply:
Good idea. I filed https://github.com/hashgraph/pbj/issues/232 to implement this at a later time.